README.md

```diff
@@ -69,12 +69,25 @@ Find **Protocol Buffers Descriptions** at the [`./pb` directory](./pb).
 
 ## Installation
 
-> **Note:** that the first build can take up to 20-30 minutes. Consequent builds
-> will be faster.
+We offer three installation methods:
+
+1. **Running locally with “Docker for Desktop”** (~20 minutes) You will build
+   and deploy microservices images to a single-node Kubernetes cluster running
+   on your development machine.
+
+2. **Running on Google Kubernetes Engine (GKE)”** (~30 minutes) You will build,
+   upload and deploy the container images to a Kubernetes cluster on Google
+   Cloud.
+
+3. **Using pre-built container images:** (You still need to follow one of the
```

Comment:
no time estimate for #3 - this on purpose?

Reply:
I'll say ~10 mins.